frontend/src/Experiment.jsx

```diff
@@ -1,12 +1,11 @@
 /* eslint-disable class-methods-use-this */
 import React, { Component } from 'react';
-import './App.css';
 
 class Experiment extends Component {
   render() {
     return (
       <div>
-        <h2>Experiment</h2>
+        <h2>Prototypes</h2>
```

Comment:
Since the title of this page is **Prototype** without an 's', would it be consistent to put **Prototype** as well here instead?

Reply:
Updated